ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLocationType.vue

```diff
@@ -218,10 +218,44 @@ import { HomeLocationTypes } from '@/enums'
 import { PidNumber } from '@/components/common'
 import HomeLocationDescription from './HomeLocationDescription.vue'
 import { useInputRules, useMhrValidations, useNewMhrRegistration } from '@/composables'
-import { FormIF, MhrLocationInfoIF } from '@/interfaces'
+import { FormIF, MhrLocationInfoIF, MhrRegistrationHomeLocationIF } from '@/interfaces'
 import { PidInfoIF } from '@/interfaces/ltsa-api-interfaces'
 import { storeToRefs } from 'pinia'
 
+const emptyLocation: MhrRegistrationHomeLocationIF = {
+    parkName: '',
+    pad: '',
+    address: {
+      street: '',
+      streetAdditional: '',
+      city: '',
+      region: null,
+      country: null,
+      postalCode: ''
+    },
+    leaveProvince: false,
+    pidNumber: '',
+    taxCertificate: false,
+    dealerName: '',
+    additionalDescription: '',
+    locationType: null,
+    otherType: null,
+    legalDescription: '',
+    lot: '',
+    parcel: '',
+    block: '',
+    districtLot: '',
+    partOf: '',
+    section: '',
+    township: '',
+    range: '',
+    meridian: '',
+    landDistrict: '',
+    plan: '',
+    bandName: '',
+    reserveNumber: '',
+    exceptionPlan: ''
+}
```

Comment:
If this isn't used anywhere but the prop default, could define in the prop.

Reply:
Yea this is just a for the prop, wanted to keep the component looking cleaner, but can certainly update it.